plugins/src/com/oracle/oci/eclipse/sdkclients/ObjStorageClient.java

```diff
@@ -198,6 +202,23 @@ public Bucket createBucket(String bucketName) {
                         .build());
         return createBucketResponse.getBucket();
     }
+    
+    public Bucket createBucket(String bucketName, String namespace) {
```

Comment:
This is not required. Set the namespace using setNamespace method and use the existing createBucket method instead of creating another one.